CONTRIBUTING.rst

```diff
@@ -209,6 +214,15 @@ The tests can be run on your local computer on all operating systems.
    cmake path/to/fish-shell
    make test
 
+Or you can run them on a fish, without involving cmake::
+
+  cargo build
+  FISHDIR=target/debug tests/test_driver.sh tests/test.fish # script tests, the checks
```

Comment:
Maybe `FISHDIR=target/debug` should be the default when not invoked by cmake.
Or perhaps `./build`. Currently it's `$PWD`, I'm not sure if that's ever useful.